conan/tools/microsoft/msbuilddeps.py

```diff
@@ -187,7 +179,7 @@ def join_paths(paths):
                     rel = full_path[len(root_folder)+1:]
                     full_path = ("%s/%s" % (pkg_placeholder, rel))
                 ret.append(full_path)
-            return "".join("{};".format(e) for e in ret)
+            return "".join("{};".format(e) for e in ret).removesuffix(";")
```

Comment:
```suggestion
            return ";".join(ret)
```

why not using the separator that handles this automatically?

Reply:
great idea.